trappy/plotter/ILinePlot.py

```diff
@@ -149,9 +149,17 @@ def __init__(self, traces, templates=None, **kwargs):
     def savefig(self, *args, **kwargs):
         raise NotImplementedError("Not Available for ILinePlot")
 
-    def view(self, test=False):
+    def view(self, max_datapoints=75000, test=False):
         """Displays the graph
 
+        :param max_datapoints: Maximum number of datapoints to plot.
+        Dygraph can make the browser unresponsive if it tries to plot
+        too many datapoints.  Chrome chokes at around 75000, Firefox
```

Comment:
I've added the processor and browser versions but I doubt it helps much.  The idea is to stop before it may get slow, the user can then decide to increase max_datapoints for the plot or reduce the size of the plot if they're interested in a certain period of time or if they are not worried about the resolution.  I wouldn't expect anybody to fine tune the number, I don't think you gain much.